### ISSUE TITLE
Improve CG and/or Stability of Monitor cart

### ISSUE THREAD

**Issue by owendelong** [socallinuxexpo/scale-network#360]
## Description
Monitor cart is too unstable front-to-back. CG either needs to be brought more inside the
box defined by the contact patch and/or lowered closer to the ground or contact patch box needs to be expanded (outriggers?).

## Acceptance Criteria
Monitor cart is stable and does not exhibit strong tipping tendency.

**Comment by davidelang** [comment]
My initial plan  is to move the shelving/back ~4" forward.

I am currently thinking to cut the back just below the bottom shelf and am debating between moving the sides forward (and trimming the front of the sides 4", then put filler pieces behind them) or re-cutting the sides

one side is only screwed on, not glued, the other side is glued, but the side was not clamped in place in a timely manner, so the glue may not have a good hold.

**Comment by sarcasticadmin** [comment]
@owendelong @davidelang Can you summarize whats transpired at the work party? That way I think we can close this out

**Comment by davidelang** [comment]
The monitor cart was cut and the top part that had the monitor shelves was moved 
forward 3" and the cart spliced back together to improve the stability.

There is a little bit of work to do, but it's minor tweaks (rounding edges and 
that sort of thing), the major work was completed.
